### PR TITLE
[DOCS] add info on initialization with `furyctl` plus historic versions and compatibility matrix informations

### PR DESCRIPTION
**Pull request by nutellinoit**
As per title, this PR covers the changes in `furyctl` v1.0.0 and adds some informations on the compatibility of KFD

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,7 +84,23 @@ Add-on modules provides additional functionality to the distribution.
...
+
+It will download an opinionated example of the `Furyfile.yml` and a `kustomization.yaml` file to your current directory.
+
+Download all the modules with:
+
+```bash
+furyctl distribution download
+```
+
+Additionally, more comprehensive examples of the KFD can be found on our [documentation site](https://docs.kubernetesfury.com/docs/distribution/#%EF%B8%8F-how-do-i-get-started).
```
[the remark this answers is not in the document]
Comment (nutellinoit) [reply README.md:103]: My point of view is to give a "quick and dirty" way to download the distribution. 
I feel that without any info on this repository is not so easy to understand what the distribution is.

[the remark this answers is not in the document]
Comment (ralgozino) [reply README.md]: Hi Samu

Regarding
> I feel that without any info on this repository is not so easy to understand what the distribution is.

In my opinion, the readme does explain what the distribution is and the philosophy behind it. The getting starting guide would be too much for the readme, that's why we point to the docs.

And regarding

> My point of view is to give a "quick and dirty" way to download the distribution.

I think that a new user would be lost with these instructions, but maybe for an advanced user (someone from our delivery team for example could find this useful), in that case, I'd suggest splitting the instructions into 2, something like:

```markdown
If you are new to KFD and want to get started, please head to the [documentation site](https://docs.kubernetesfury.com/docs/distribution/#%EF%B8%8F-how-do-i-get-started)

If you are an advanced user, you can get started by using [`furyctl`][furyctl] to download a `Furyfile.yml` and a `kustomization.yaml` template file:

````sh
furyctl distribution template --version v1.23.1
``` // I don't know how to escape markdown inside a comment :D  

Edit the files accordingly to your needs and then download the modules by running:

```sh
furyctl distribution download
``` // I don't know how to escape markdown inside a comment :D  

```

What do you think?

[the remark this answers is not in the document]
Comment (nutellinoit) [reply README.md]: I like the approach to change the paragraph like you pointed out, i will make the changes now

File: README.md [hunk 2]
```diff
@@ -72,7 +84,20 @@ Add-on modules provides additional functionality to the distribution.
...
+
+```bash
+furyctl distribution template --version v1.23.1
+```
+
+Edit the files accordingly to your needs and then download the modules by running:
+
+```bash
+furyctl distribution download
+```
```
Comment (ralgozino) [inline README.md:101]: I think we are missing a call to action here at the end, I suggest adding something like:

```suggestion

You can find now the modules manifests in the `vendor` folder and start working with them, refer to each module's documentation for further details.

```

**Review by ralgozino — APPROVED** [review]
Approving with just a minor suggestion.

We'll need to hold the merge of this PR until furyctl v1 (with the new command structure) has been released

**Comment by nutellinoit** [comment]
Yes, we should leave this PR as approved and ready to be merged

**Pull request closed without merging**